### PR TITLE
changes behavior of obsAtFront()

### PR DESCRIPTION
**Pull request by Druner**
#47 changes the behavior when obsAtFront() so that the car reverses instead of just stops. Changes angles of front right and front left sensors to be wider. These changes improves autoRightPark() to work in more scenarios.

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

File: arduino/smartcar/smartcar.ino [hunk 1]
```diff
@@ -259,42 +264,47 @@ void autoRightPark(){ // the car is supposed to park inside a parking spot to it
     car.setSpeed(15);
     Serial.println(targetAngle);
     Serial.println(currentAngle);
-    Serial.println(rightTimer);
-    while (targetAngle < currentAngle){
+    while (targetAngle <= currentAngle){
```
Comment (adityak714) [inline arduino/smartcar/smartcar.ino:267]: what does this specifically do?

**Review by sh4r10 — APPROVED** [review]
Looks good

**Comment by arrestoteles** [comment]
@Druner we should use verbs when naming methods.

**Comment by Druner** [comment]
> @Druner we should use verbs when naming methods.

Bit late but got fixed here: https://github.com/DIT113-V22/group-02/commit/64084f8bcf40ed0f1f62002418851c25ae20744e

**Pull request merged**